util/util.go

```diff
@@ -184,3 +185,29 @@ func InstallArchive(ctx context.Context, rawURL string, dest string) error {
 	}
 	return nil
 }
+
+func getTiDBReplicaRead(job string, db *sql.DB) string {
+	var replicaRead string
+	if err := db.QueryRow("select @@tidb_replica_read").Scan(&replicaRead); err != nil {
+		log.Fatalf("[%s] get tidb_replica_read fail: %v", job, err)
+	}
+	return replicaRead
+}
+
+func setTiDBReplicaRead(job string, db *sql.DB, mode string) {
+	sql := fmt.Sprintf("set @@tidb_replica_read = '%s'", mode)
+	if _, err := db.Exec(sql); err != nil {
+		log.Fatalf("[%s] tidb_replica_read set fail: %v", job, err)
+	}
+}
+
+// RandomlyChangeReplicaRead changes `tidb_replica_read` randomly in a session.
+func RandomlyChangeReplicaRead(job, replicaRead string, db *sql.DB) {
```

Comment:
IMO, Putting these three methods here isn't suitable.

Reply:
They are not only for bank/bank2. All other cases can use them.

Reply:
OK, but "bank2" is dead coded here. :)

Reply:
sorry, don't see it.